instance/github.py

```diff
@@ -158,13 +159,8 @@ def __init__(self, number, source_fork_name, target_fork_name, branch_name, titl
         self.title = title
         self.username = username
         self.body = body
-
-    @property
-    def extra_settings(self):
-        """
-        Extra settings contained in the PR body
-        """
-        return get_settings_from_pr_body(self.body)
+        self.extra_settings = get_settings_from_pr_body(body)
+        self.extra_settings_dict = yaml.load(self.extra_settings) or {}
```

Comment:
@antoviaque I had already changed this before I saw your last comment.  We should get this PR out, so we can use swap by default.  I'll prepare a new small PR with a suggestion how to make this nicer, if you are fine with committing the latest version of this PR.